content/quickstart.md

```diff
@@ -7,29 +8,72 @@ tocopen: true
 
 # Prerequisites
 
-Under construction.
+1- OpenSeeFace tracking: webcam
 
-# Installation
+2- IFacialMocap tracking: iOS device with face ID or A12 (or newer) chip
+
+3- MeowFace tracking: android device that supports ARCore (most modern phones do)
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+4- 64-bit Operating system with x86 architecture

```

Comment:
Technically wrong, it should be x86_64 (intel) or amd64 (amd). x86 is 32 bit. Would be better to leave out the architecture entirely or leave it vague at "Any mainstream 64-bit operating system"